### ISSUE TITLE
weird vg sim usage in jenkins/vgci.py::VGCITest::test_sim_mhc_snp1kg_mpmap

### ISSUE THREAD

**Issue by ekg** [vgteam/vg#1615]
I am trying to debug why test_sim_mhc_snp1kg_mpmap and friends have been breaking in https://github.com/vgteam/vg/pull/1612. Today I spent a lot of time running:

```
jenkins/jenkins.sh -l -r -i -k -s -t jenkins/vgci.py::VGCITest::test_sim_mhc_snp1kg_mpmap
```

I notice that it seems to be running a number of small sim jobs, each producing ~1500 reads. Is this to be expected?

```
2018-04-09 17:18:02,460 - toil-rt - INFO - Run: vg sim -x index.xg -n 1562 -d 0.01 -p 1000 -v 75.0 -S 5 -I -s 11 -F error_template.fastq -a | tee sim_0_3.gam | vg annotate -p -x annot_index.xg -a - | tee sim_0_3_annot.gam | vg view -aj -
```

It seems rather inefficient to me to do this, and it's taking a while (which could be the bug I'm chasing). In any case I found the design odd.

**Comment by adamnovak** [comment]
It's helpful to break up sim into multiple chunks when building really big GAMs. We might be using too many chunks in this case.

I think it's controlled by a toil-vg config parameter, so we could update the test to use fewer chunks and be faster.

**Comment by glennhickey** [comment]
Yeah, it's using the same chunking for all the tests.  Never noticed it
being a bottleneck on the smaller ones, but it should be easy to
parameterize
